bdb/file.c

```diff
@@ -3169,21 +3169,24 @@ if (!is_real_netinfo(bdb_state->repinfo->netinfo))
         }
     }
 
```

Comment:
This is one of the initial "unable to start a cluster on lxc" issues that Kyle found - on the first day of the first week that he arrived.  The previous code incorrectly assumed that the current node cannot become master inside of this loop.  In a case I observed on my own lxc cluster, a machine managed to become master after the original check (line 3179).  net_send_message fails automatically for self-sent messages, forcing this to sleep and loop 'startup_sync_attempts' times before finally exiting.